lms/static/certificates/sass/_components.scss

```diff
@@ -0,0 +1,641 @@
+    background: palette(grayscale, white-t);
+    padding: spacing-vertical(small) spacing-horizontal(base);
+}
+
+%depth-well {
+    box-shadow: 0 rem(1) rem(3) 0 palette(grayscale-cool, light) inset;
+    padding: spacing-vertical(small) spacing-horizontal(base);
+    background: palette(grayscale, x-light);
+}
+
+%depth-card {
+    box-shadow: 0 rem(1) rem(2) 0 palette(grayscale-cool, light);
+    padding: spacing-vertical(small) spacing-horizontal(base);
+    background: palette(grayscale, white-t);
+}
+
+.depth-base-focus {
+    @extend %depth-base-focus;
+}
+
+.depth-well {
+    @extend %depth-well;
+}
+
+.depth-card {
+    @extend %depth-card;
+}
+
+
+// ------------------------------
+// #IMAGES
+// ------------------------------
+// circular cropped
+%img-cropped {
+
+    .src {
+        display: block;
+        width: 100%;
+        border-radius: 50%;
+    }
+}
+
+// ------------------------------
+// #HEADER APP
+// ------------------------------
+.wrapper-header {
+    @extend %divider-2;
+    border-bottom-color: palette(grayscale, xx-trans);
+    background: palette(grayscale, white-t);
+}
+
+.header-app {
+
+    .logo {
+        @extend %reset-link;
+    }
+}
+
+// ------------------------------
+// #BANNER
+// ------------------------------
+.banner {
+
+    .message-copy {
+        margin-bottom: 0;
+    }
+}
+
+// message to user-centric banner
+.wrapper-banner-user {
+    box-shadow: inset 0 rem(2) rem(2) 0 palette(grayscale, black-t);
+    background: palette(grayscale-cool, x-dark);
+}
+
+.banner-user {
+
+    .message-title {
+        color: palette(grayscale, white-t);
+        font-weight: font-weight(semi-bold);
+    }
+
+    .message-copy {
+        color: palette(grayscale, white);
+    }
+
+    .message-actions .action {
+        display: inline-block;
+        vertical-align: middle;
+        margin-right: spacing-horizontal(small);
+        font-size: font-size(mid-large);
+
+        &:last-child {
+            margin-right: 0;
+        }
+    }
+}
+
+// ------------------------------
+// #INTRODUCTION
+// ------------------------------
+.introduction {
+    margin-bottom: spacing-vertical(base);
+    text-align: center;
+}
+
+// ------------------------------
+// #ACCOMPLISHMENTS
+// ------------------------------
+// base accomplishment
+.accomplishment {
+
+    .accomplishment-type-symbol {
+        @extend %img-cropped;
+    }
+}
+
+// main accomplishment
+.accomplishment-main {
+    background: palette(grayscale-cool, x-trans);
+}
+
+// brief accomplishment
+.accomplishment-brief {
+    @extend %depth-card;
+
+    .accomplishment-type-symbol, .accomplishment-details {
+        display: inline-block;
+        vertical-align: middle;
+    }
+
+    .accomplishment-type-symbol {
+        @include size(20%);
+    }
+
+    .accomplishment-details {
+        width: 70%;
+    }
+
+
+    .accomplishment-summary {
+        margin-bottom: 0;
+    }
+}
+
+// ------------------------------
+// #RENDERING
```

Comment:
NOTE: I think the plan is to not have any accomplishments listed on these pages in the short term. It isn't currently a Saudi commitment and it isn't clear when we would revisit supplemental accomplishments on these views. I suppose upon reflection it doesn't hurt us to have it in the FED / styling code early. 

Q: One note about this abstraction is also that this would eventually (presumably?) move out of the certificates application context if we expect multiple parts of the platform to use it. I would expect this to move to the /openedx/ folder (our new version of common), or whatever the new PL work suggests from a file organization standpoint. 

(No changes necessary here, just commenting on the new and forward thinking work her! :100: )